frontend/public/components/build-config.tsx

```diff
@@ -309,13 +364,34 @@ export const BuildConfigsPage: React.FC<BuildConfigsPageProps> = (props) => {
     { id: BuildStrategyType.Custom, title: t('public~Custom') },
   ];
 
+  const statusFilters = [
+    { id: 'New', title: t('public~New') },
+    { id: 'Pending', title: t('public~Pending') },
+    { id: 'Running', title: t('public~Running') },
+    { id: 'Complete', title: t('public~Complete') },
+    { id: 'Failed', title: t('public~Failed') },
+    { id: 'Error', title: t('public~Error') },
+    { id: 'Cancelled', title: t('public~Cancelled') },
+    { id: 'Unknown', title: t('public~Unknown') },
+  ];
+
   const filters = [
     {
       filterGroupName: t('public~Build strategy'),
       type: 'build-strategy',
       reducer: buildStrategy,
       items: allStrategies,
     },
+    {
+      filterGroupName: t('public~BuildRun status'),
```

Comment:
nit: For BuildConfigs the status is loaded from a Build, not a BuildRun:

```suggestion
      filterGroupName: t('public~Build status'),
```